packages/trader/src/AppV2/Containers/Trade/trade.scss

```diff
@@ -67,10 +67,6 @@
 
         &--with-button {
             min-height: 13.6rem;
-
-            &:has(.trade-params__options__wrapper--horizontal:not(:only-child)) {
```

Comment:
This style was needed for cases, when purchase btn is absent (e.g. when the market is closed). Please, check

Reply:
Works as expected, thank you